libcipher/caesar.py

```diff
@@ -0,0 +1,64 @@
+__author__ = 'kskrishnasangeeth'
+
+import random
+import sys
+
+
+def encrypt(message, key=None):
+    """Returns the encrypted messsage for the string passed as argument.
+
+    Uses a random key for generating the cipher text
+
+    Args:
+        message (str) : The message to be encrypted.
+        key (int) : The key based on which encryption is performed. Is optional value.
+
+    Return:
+        str: the encrypted message """
+
+    if key is None:
+        key = random.randint(1, 25)
+
+    if key <= 0 or key >= 27:
+        print('Key should be having value between 1 and 26 ')
+        sys.exit(0)
+    cipher_text = ""
+    message = message.lower()
+    for each_char in message:
+        if each_char == ' ' or not each_char.isalpha():
+            cipher_text += each_char
+            continue
+        else:
+            changed_char = chr(ord(each_char) + key)
+            if ord(changed_char) > ord('z'):
+                changed_char = chr(ord(each_char) + (key - 26))
+            cipher_text += changed_char
+    return cipher_text
+
+
+def decrypt(message, key):
```

Comment:
Both `encrypt()` and `decrypt()` are identical. Can we use a core private method that does both and implement the encrypt and decrypt as wrappers of it?